sbt-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -114,14 +147,16 @@ object ScalaNativePluginInternal {
     val flags   = Seq("-o", outpath) ++ linkopts ++ opts
     val compile = abs(clangpp) +: (flags ++ paths)
 
-    Process(compile, target).!
+    logger.info(running(compile))
+
+    Process(compile, target) ! logger
   }
 
   lazy val projectSettings = Seq(
     libraryDependencies ++= Seq(
-      "org.scala-native" %% "nativelib" % nativeVersion,
-      "org.scala-native" %% "javalib"   % nativeVersion,
-      "org.scala-native" %% "scalalib"  % nativeVersion
+      "org.scala-native" %%% "nativelib" % nativeVersion,
+      "org.scala-native" %%% "javalib"   % nativeVersion,
+      "org.scala-native" %%% "scalalib"  % nativeVersion
```

Comment:
No, only that they have their `crossVersion` set up to add the `_native` suffix (or whatever that one is).

FTR, however, the equivalents in Scala.js do *not* have their `crossVersion` set up like that. So `scalajs-library.jar` is only named `scalajs-library_2.11.jar`, and is referred to with `%%`. I felt that made sense wrt. the fact that `scala-library.jar` is not cross-versioned at all, and is referred to as `%`.